deepmd/utils/network.py

```diff
@@ -44,19 +44,31 @@ def one_layer(inputs,
                             w_initializer, 
                             trainable = trainable)
         variable_summaries(w, 'matrix')
-        b = tf.get_variable('bias', 
-                            [outputs_size], 
-                            precision,
-                            b_initializer, 
-                            trainable = trainable)
-        variable_summaries(b, 'bias')
+        if final_layer and GLOBAL_ASCEND_OUT_PRECISION:
```

Comment:
@denghuilu It looks similar as `mixed_prec`. However the weight for `mixed_prec` is cast but the weight for `GLOBAL_ASCEND_OUT_PRECISION` is not cast. What do you think about it?

Reply:
It seems like they have already set the precisions before running the networks:https://github.com/ZhengdQin/deepmd-kit/blob/58367bea72eb31c9797373bbb9f93e6bbef1aa78/deepmd/utils/transfer_to_ascend.py#L77-L78.

Reply:
Thanks, we modify the original precision directly. We tried different mix-precision models on Ascend platform and found that the GLOBAL_ASCEND_OUT_PRECISION being float32 (only the last biasadd is float32) is important to ensure the accuracy of the ascend transfered model, so we cast the every weight except the last biasadd.

Reply:
It has the same aim as `mixed_prec` so it's better to merge these two variables.

Reply:
Thanks for the comment! Considering our mix-precision model is defferent with the mixed_prec defined model. only the last biasadd is the float32 type, so using mixed_prec needs to change the code logic and makes it difficult to understand.